apps/namadillo/src/App/Masp/MaspShield.tsx

```diff
@@ -26,9 +26,10 @@ import { Address, TransferTransactionData } from "types";
 export const MaspShield: React.FC = () => {
   const navigate = useNavigate();
   const [searchParams, setSearchParams] = useSearchParams();
+  const { storeTransaction } = useTransactionActions();
   const [displayAmount, setDisplayAmount] = useState<BigNumber | undefined>();
   const [generalErrorMessage, setGeneralErrorMessage] = useState("");
-
+  const [currentStep, setCurrentStep] = useState("");
```

Comment:
Can you apply this on MaspUnshield and NamadaTransfer as well? Or is this part of a future PR?

Reply:
Hello! This is going to be part of a PR I'm working now. We're going to change the whole interface after submitting the transaction, that's why I didn't apply this change to other pages